project.md

```diff
@@ -11,41 +11,79 @@ _What need is met by your idea? Who are you helping? What is that person's
+
+
 ### Operation (semantics)
 
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+While I haven’t finalized the syntax/semantics of the DSL yet, I would like to use an external DSL using parser combinators to read in the user input. The program the user writes would then be read, and then this could either result in an Icalendar ICS file, or it could be passed to a second DSL which will visualize it in a HTML/Excel file. 
+	Some errors that might occur could be type errors, such as inputting “29:72pm” for the start time, or syntax errors such as not giving a title for the event. I think this could be avoided by simply catching and returning errors for these. These could then be communicated to the user via error messages. I know there are probably more errors that are possible, but I can not think of any because it is 3:15 am and I am very tired. 
+
+
 ### Expressiveness
 
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+I want it to be very very easy to create simple events. This should take less than 20 seconds max. I also want these events to be output as a file which could then be imported to another calendar app. This should be easy by limiting the number of arguments to pass to the DSL.
+	I think it should be possible, but difficult to write very long, descriptive events. I think with this DSL I want to prioritize speed/ease of use, and if a user is trying to write a very long title or description I think writing that in a text file or terminal might be difficult, but still possible.
+	It should be impossible to take advantage of much of the features offered by calendar apps. This would include google meet links, different types of events like tasks or focus time or out of office, and calendar event ownership. The user could import the ICS file output into a calendar app and then add these new features, but I think I would want to at least limit this in the DSL. 
+
```

Comment:
This sounds very extreme - but also exactly what you want to do - really narrow in what a "calendar/scheduler" tool needs to do at the bare minimum, and do that as efficiently as possible. I feel like anyone outside of your audience would Hate this, but yourself and anyone else looking for the same functionality would Love it.